translator/translate/logs/logs_test.go

```diff
@@ -18,8 +18,10 @@ import (
 )
 
 func TestLogs(t *testing.T) {
+
```

Comment:
Why do we need to add this newline here?

Reply:
Not needed probs just some weird formatting from intelij.